packages/suite-native-settings/src/components/DeviceSettings.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { SettingsSection } from './SettingsSection';
+import { SettingSectionItem, SettingsSectionItem } from './SettingsSectionItem';
+
+const deviceSettingItems: SettingSectionItem[] = [
+    { title: 'Customization', description: 'Name, Homescreen', iconName: 'palette' },
+    { title: 'Security', description: 'PIN Active, Passphrase enabled', iconName: 'lock' },
+    { title: 'Danger Area', description: 'Factory reset, Custom firmware', iconName: 'lock' },
+];
+
+export const DeviceSettings = () => (
+    <SettingsSection title="Device">
+        {deviceSettingItems.map(({ title, iconName, description }: SettingSectionItem) => (
+            <SettingsSectionItem
+                key={title}
+                iconName={iconName}
```

Comment:
I would really prefer just spread `{...settingsItem}` here, no reason to repeat everything

Or just define it manually without array

Reply:
[8c23b9f](https://github.com/trezor/trezor-suite/pull/5617/commits/8c23b9f9cafd67684e4e1e0e3c5c399d53c6bc34)